src/main/java/com/acme/conferencesystem/ticket/persistence/TicketEntity.java

```diff
@@ -0,0 +1,19 @@
+package com.acme.conferencesystem.ticket.persistence;
+
+import com.acme.conferencesystem.ticket.business.TicketCategory;
+import com.acme.conferencesystem.ticket.business.TicketStatus;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.relational.core.mapping.Table;
+import org.springframework.format.annotation.DateTimeFormat;
+
+import java.time.LocalDate;
+import java.util.UUID;
+
+@Table("tickets")
+public record TicketEntity(@Id UUID id,
+                           TicketCategory category,
+                           @DateTimeFormat(pattern = "yyyy-MM-dd HH:mm:ssX")
+                           LocalDate date,
```

Comment:
You should use LocalDateTime if you want to store the time. I don't think you need the annotation.

Reply:
Look how I did it in proposals. Even if I realized that I have a repeated functionality.

Creating the date in the database with 
creation_date_time TIMESTAMP DEFAULT CURRENT_TIMESTAMP

and also with annotations in the ProposalEntity.

Reply:
This was only a test, because I was receiving serialization errors as the repository is persisting the complete time format when I'm only storing the date. I'll keep working on this